src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -41,10 +43,14 @@
     public void before() throws IOException {
         // create event-type and subscribe to it
         eventType = createEventType();
-        subscription = createSubscription(ImmutableSet.of(eventType.getName()));
+        final SubscriptionBase subscription = randomSubscription()
+                .withEventType(eventType.getName())
+                .withStartFrom(BEGIN)
+                .buildSubscriptionBase();
+        this.subscription = createSubscription(subscription);
     }
 
-    @Test(timeout = 30000)
```

Comment:
I thought that RestAssured has default timeout :) and I think it is better to set it